_includes/footer_scripts.html

```diff
@@ -1,8 +1,10 @@
-<script src="{{ site.baseurl }}/assets/mobile-menu.js"></script>
-<script src="{{ site.baseurl }}/assets/scroll-to-anchor.js"></script>
-<script src="{{ site.baseurl }}/assets/external-links-new-tab.js"></script>
-<script src="{{ site.baseurl }}/assets/search-bar.js"></script>
-<script src="{{ site.baseurl }}/assets/cookie-banner.js"></script>
+  <script src="{{ site.baseurl }}/assets/mobile-menu.js"></script>
+  <script src="{{ site.baseurl }}/assets/scroll-to-anchor.js"></script>
+  <script src="{{ site.baseurl }}/assets/external-links-new-tab.js"></script>
+  {% if page.deep-learning-index != true %}
+  <script src="{{ site.baseurl }}/assets/search-bar.js"></script>
+  {% endif %}
+  <script src="{{ site.baseurl }}/assets/cookie-banner.js"></script>
```

Comment:
I don't think you have to indent these scripts--except for the one inside of the conditional.